pkg/utils/utils_test.go

```diff
@@ -696,7 +696,12 @@ var _ = Describe("Utils", Label("utils"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			Expect(size).To(Equal(int64(3072)))
 		})
-		It("Returns the expected size of a test folder", func() {
+		It("Returns the size of a test folder when skipping subdirectories", func() {
```

Comment:
Nice, loving this sort of tests. They are super useful in case someday in the future we change some apparently simple utility.